antarest/study/business/binding_constraint_management.py

```diff
@@ -658,34 +677,31 @@ def update_binding_constraint(
         data: ConstraintInput,
     ) -> ConstraintOutput:
         file_study = self.storage_service.get_storage(study).get_raw(study)
-        existing_constraint = self.get_binding_constraint(study, ConstraintFilters(bc_id=binding_constraint_id))
+        existing_constraint = self.get_binding_constraint(study, binding_constraint_id)
         study_version = int(study.version)
-        if not isinstance(existing_constraint, ConstraintOutputBase) and not isinstance(
-            existing_constraint, ConstraintOutput870
-        ):
-            raise BindingConstraintNotFoundError(study.id)
-
         check_attributes_coherence(data, study_version)
 
         # Because the update_binding_constraint command requires every attribute we have to fill them all.
         # This creates a `big` command even though we only updated one field.
         # fixme : Change the architecture to avoid this type of misconception
-        updated_constraint = {
+        upd_constraint = {
```

Comment:
in my opinion, it's better not to use this kind of abbreviation, as it's less comprehensible. It's better to be more explicit, even if it means taking up a little more space.

Reply:
(it as meaning for you in the given moment but another developer might not get it later on)